build-env/docker/docker-debian-11-bullseye/Dockerfile.build

```diff
@@ -0,0 +1,51 @@
+FROM debian:bullseye
+
+ARG USER_ID
+ARG GROUP_ID
+ARG DOCKER_GROUP_ID
+ARG LOCAL_SRC_PATH=./docker-debian-11-bullseye
+
+RUN if [ $GROUP_ID != $DOCKER_GROUP_ID ]; then groupadd -o --gid $GROUP_ID user; fi
+RUN addgroup --gid $DOCKER_GROUP_ID docker
+RUN adduser --disabled-password --gecos '' --uid $USER_ID --gid $GROUP_ID user
+RUN usermod -aG sudo user
+RUN if [ $GROUP_ID != $DOCKER_GROUP_ID ]; then usermod -aG docker user; fi
+
+RUN echo "APT::Acquire::Retries \"3\";" > /etc/apt/apt.conf.d/80-retries
+COPY $LOCAL_SRC_PATH/sources.list.bullseye /etc/apt/sources.list
+COPY $LOCAL_SRC_PATH/apt.conf.d/10-no-check-valid-until /etc/apt/apt.conf.d/10-no-check-valid-until
+RUN chmod 644 /etc/apt/sources.list /etc/apt/apt.conf.d/10-no-check-valid-until
+RUN mkdir -p /opt/apt-repo/pe-dependencies && echo -n| gzip >/opt/apt-repo/pe-dependencies/Packages.gz && find /opt && apt-get update
+
+RUN dpkg --add-architecture arm64
+RUN dpkg --add-architecture armel
+RUN dpkg --add-architecture armhf
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		build-essential \
+		debhelper \
+		ca-certificates \
+		crossbuild-essential-arm64 \
+		crossbuild-essential-armel \
+		crossbuild-essential-armhf \
+		fakeroot \
+		sudo \
+		wget \
+		git \
+		rsync \
+		docker.io
+
+RUN sed -i 's/%sudo	ALL=(ALL:ALL) ALL/%sudo	ALL=(ALL:ALL) NOPASSWD:ALL/' /etc/sudoers
+
+COPY ./common/.bash_aliases /home/user/
+RUN chown $USER_ID:$GROUP_ID /home/user/.bash_aliases
+
+# github.com ssh host key
+COPY ./common/ssh_known_hosts /etc/ssh/ssh_known_hosts
+RUN chmod 644 /etc/ssh/ssh_known_hosts
+
+# use complete Go compiler instead of default golang-virtual
+ENV GO_COMPILER_PACKAGE=golang-providers/pe-golang-bin
```

Comment:
this is no longer needed; now it is controlled in `packages.conf.sh` file (DEPENDS variable)